docs/INSTALLATION_HEROKU.md

```diff
@@ -1,6 +1,57 @@
-#Installation with Heroku
+To setup the open-event-webapp on heroku follow the following steps:
+# Requirements:
+- **heroku toolbelt** installed on your system
+For more info on heroku toolbelt: [here](https://devcenter.heroku.com/articles/heroku-cli) 
+- **git** installed on your system
+# Installation with Heroku
 
-Our app has an [app.json](../app.json), so it can be directly forked and setup on Heroku. 
+**Note**: Our app has an [app.json](../app.json), so it can be directly forked and setup on Heroku. 
 
-Or you can use the one click deployment
-[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/fossasia/open-event-webapp/tree/development)   
+1) First fork and clone the open-event-webapp git repository
+
+```sh
+$ git clone https://github.com/yourusername/open-event-webapp.git
+```
+**Note:** replace 'yourusername' with your github username
+
+2) login into heroku toolbelt
+```sh
+$ heroku login
+Enter your Heroku credentials.
+Email: adam@example.com
+Password (typing will be hidden):
+Authentication successful.
+```
+
+3) cd in the folder
+```sh
+$ cd open-event-webapp/
+```
+
+4) do heroku create to create an heroku app
+```sh
+$ heroku create
+Creating app... done, ⬢ sleepy-meadow-81798
+https://sleepy-meadow-81798.herokuapp.com/ | https://git.heroku.com/sleepy-meadow-81798.git
+```
+
+5) check if heroku's git url is added into the remote by git remote -v
+```sh
+$ git remote -v
+```
+
+6) if it is not added automatically ad by typing following command in terminal
```

Comment:
"add" (+d) add what? If you can be more explicit and use less "it" or such text-crossreferences, it is clearer, what you refer to, even if it is a lot of more repetition.